src/bindings/python/src/openvino/__init__.py

```diff
@@ -75,12 +90,24 @@
 from openvino import opset15
 from openvino import opset16
 
+# Extend Node class to support binary operators
+Node.__add__ = opset13.add
+Node.__sub__ = opset13.subtract
+Node.__mul__ = opset13.multiply
+Node.__div__ = opset13.divide
+Node.__truediv__ = opset13.divide
+Node.__radd__ = lambda left, right: opset13.add(right, left)
+Node.__rsub__ = lambda left, right: opset13.subtract(right, left)
+Node.__rmul__ = lambda left, right: opset13.multiply(right, left)
+Node.__rdiv__ = lambda left, right: opset13.divide(right, left)
+Node.__rtruediv__ = lambda left, right: opset13.divide(right, left)
```

Comment:
Why do we have `opset13` here specifically? I'm not blocking the PR, it's a separate discussion.

There's no mechanism to get the "newest opset" op (which I think would be a good addition to the API), but still, there has to be room for improvement here. We may introduce a new version for one of these ops, for any reason, it's an edge case to remember that it also should be bumped here.

Reply:
Moreover, these arithmetic methods are semantically incorrect to have. That is because we can't sum up nodes that have more one outputs.
It is more logical to define such methods for ov::Output (corresponding to tensors).